tests/integration/components/rive-animation-test.js

```diff
@@ -0,0 +1,176 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'codecrafters-frontend/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+// Mock Rive class for testing
+class MockRive {
```

Comment:
In general this seems to be a pretty clumsy testing strategy - we don't get a lot of benefit out of it. I'd recommend dropping this mocking entirely and just checking that (a) there are no errors and (b) some kind of assertion that the canvas has things drawn inside it (i.e. an "end-to-end" test)